Spark_practice/spark_task_1.py

```diff
@@ -0,0 +1,133 @@
+                                    t.StructField('endYear', t.IntegerType(), True),
+                                    t.StructField('runtimeMinutes', t.IntegerType(), True),
+                                    t.StructField('genres', t.StringType(), True)])
+
+title_basics_sp = spark.read.csv('/home/kirill/PycharmProjects/spark_task_data/title_basics.tsv',
+                                 sep='\t',
+                                 header=True,
+                                 nullValue='\\N',
+                                 schema=title_basics_schema)
+
+title_episode_schema = t.StructType([t.StructField('tconst', t.StringType(), True),
+                                     t.StructField('parentTconst', t.StringType(), True),
+                                     t.StructField('seasonNumber', t.IntegerType(), True),
+                                     t.StructField('episodeNumber', t.IntegerType(), True)
+                                     ])
+
+title_episode_sp = spark.read.csv('/home/kirill/PycharmProjects/spark_task_data/title_episode.tsv',
+                                  sep='\t',
+                                  header=True,
+                                  nullValue='\\N',
+                                  schema=title_episode_schema)
+
+title_principals_schema = t.StructType([t.StructField('tconst', t.StringType(), True),
+                                        t.StructField('ordering', t.StringType(), True),
+                                        t.StructField('nconst', t.StringType(), True),
+                                        t.StructField('category', t.StringType(), True),
+                                        t.StructField('job', t.StringType(), True),
+                                        t.StructField('characters', t.StringType(), True)
+                                        ])
+
+title_principals_sp = spark.read.csv('/home/kirill/PycharmProjects/spark_task_data/title_principals.tsv',
+                                     sep='\t',
+                                     header=True,
+                                     nullValue='\\N',
+                                     schema=title_principals_schema)
+
+title_ratings_schema = t.StructType([t.StructField('tconst', t.StringType(), True),
+                                     t.StructField('averageRating', t.FloatType(), True),
+                                     t.StructField('numVotes', t.IntegerType(), True)
+                                     ])
+
+title_ratings_sp = spark.read.csv('/home/kirill/PycharmProjects/spark_task_data/title_ratings.tsv',
+                                  sep='\t',
+                                  header=True,
+                                  nullValue='\\N',
+                                  schema=title_ratings_schema)
+
+path_to_save = '/home/kirill/PycharmProjects/python_education_backub052122/Spark_practice/output/'
```

Comment:
absolute path shouldn't be present in the code

Reply:
done